tests/Webapi/Dom/Webapi__Dom__EventTarget__test.res

```diff
@@ -27,3 +27,16 @@ let customEvent = CustomEvent.makeWithOptions(
   },
 )
 let _ = target->dispatchEvent(customEvent)
+
+/* dispatch custom event with typed detail */
+module Detail = {
+  type t = {test: string}
+}
+
+let t: Detail.t = {
+  test: "test",
+}
```

Comment:
Can this be done without the `let t`? I'm looking at the generated JS and it would be nice to avoid that part.

Reply:
I guess you mentioned this generated js part
```js
var t = {
  test: "test"
};
```
I'm afraid that the omission of this `let t` can't be made unless we change the test case to `make` instead of `makeWithOption`. or renaming the `let t` would be nicer such as `option` maybe?

Reply:
But why does it need to be  separate `let`? It's just a simple record, can't it be `EventWithDetail.makeWithOptions("event-with-detail", { test: "test" })` ?

Reply:
Oh, I misunderstood your intention. I removed it and pushed it.

Reply:
There was two of them - sorry, please make that change to both :)

Reply:
Ah! I missed that, thx. I fixed it and pushed it. I reworded commit messages also.